modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormViewFormInstanceRecordsDisplayContext.java

```diff
@@ -605,19 +595,16 @@ private List<String> _getOptionsRenderedFormFieldValues(
 		DDMFormFieldOptions ddmFormFieldOptions,
 		List<String> renderedFormFieldValues) {
 
-		Stream<String> stream = renderedFormFieldValues.stream();
+		List<String> values = new ArrayList<>();
 
-		List<String> convertedFormFieldValues = stream.flatMap(
-			renderedFormFieldValue -> Arrays.stream(
-				StringUtil.split(renderedFormFieldValue, CharPool.COMMA))
-		).map(
-			String::trim
-		).collect(
-			Collectors.toList()
-		);
+		for (String renderedFormFieldValue : renderedFormFieldValues) {
+			Collections.addAll(
+				values,
+				StringUtil.split(renderedFormFieldValue, CharPool.COMMA));
+		}
 
 		return ListUtil.toList(
-			convertedFormFieldValues,
+			TransformUtil.transform(values, String::trim),
```

Comment:
@LisztLiszt ListUtil.toList does something similar or same with TransformUtil.transform. Please try to merge these into one TransformUtil.transform call.